zenodo/modules/records/serializers/__init__.py

```diff
@@ -87,6 +88,8 @@
 citeproc_v1 = CiteprocSerializer(csl_v1)
 #: OpenAIRE JSON serializer
 openaire_json_v1 = JSONSerializer(RecordSchemaOpenAIREJSON, replace_refs=True)
+#: JSON-LD serializer
+schemaorg_jsonld_v1 = JSONSerializer(CreativeWorkV1, replace_refs=True)
```

Comment:
we're actually using regular serializer since there were problems with getting the correct serialized output from JSONLD one, and it was also making a request to compact/expand the json